### PR TITLE
Lo-Fi Wireframing Kit for quick UX design sketching & wireframing

### PR DESCRIPTION
**Pull request by alexlazovic**

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -323,5 +323,18 @@
...
-  }
+  },
+  {
+      "name": "Lo-Fi Wireframing Kit",
+      "description": "A collection of lo-fi wireframing components.",
+      "authors": [
+        {
+          "name": "Aleksandra Lazovic",
+          "url": "https://spfr.co"
+        }s
```
Comment (lipis) [inline libraries.json:334]:
```suggestion
        }
```

File: libraries.json [hunk 2]
```diff
@@ -323,5 +323,18 @@
...
+      ],
+      "source": "spfr/Lo-Fi_Wireframing_Kit.excalidrawlib",
+      "preview": "spfr/Lo-Fi_Wireframing_Kit.png",
+      "date": "2021-02-10"
+    }
```
Comment (lipis) [inline libraries.json]:
```suggestion
  {
    "name": "Lo-Fi Wireframing Kit",
    "description": "A collection of lo-fi wireframing components.",
    "authors": [
      {
        "name": "Aleksandra Lazovic",
        "url": "https://spfr.co"
      }
    ],
    "source": "spfr/Lo-Fi_Wireframing_Kit.excalidrawlib",
    "preview": "spfr/Lo-Fi_Wireframing_Kit.png",
    "date": "2021-02-10"
  }
```

File: libraries.json [hunk 3]
```diff
@@ -323,5 +323,18 @@
...
+      "name": "Lo-Fi Wireframing Kit",
+      "description": "A collection of lo-fi wireframing components.",
+      "authors": [
+        {
+          "name": "Aleksandra Lazovic",
+          "url": "https://spfr.co"
+        }
+      ],
+      "source": "spfr/Lo-Fi_Wireframing_Kit.excalidrawlib",
+      "preview": "spfr/Lo-Fi_Wireframing_Kit.png",
```
Comment (lipis) [inline libraries.json:337]: can we rename them to use lower case?
```suggestion
      "source": "spfr/lo-fi-wireframing-kit.excalidrawlib",
      "preview": "spfr/lo-fi-wireframing-kit.png",
```

Also you'll need to rename the actual files.

**Review by lipis — APPROVED** [review]

**Review by ad1992 — APPROVED** [review]
Looks great 🎉

**Comment by dwelle** [comment]
Thanks @alexlazovic!

**Pull request merged**